snmp/datadog_checks/snmp/data/default_profiles/_generic-ucd.yaml

```diff
@@ -0,0 +1,136 @@
+metrics:
+  - MIB: UCD-SNMP-MIB
+    symbol:
+      name: cpu.usage
+      OID: 1.3.6.1.4.1.2021.10.1.5.1
+  - MIB: UCD-SNMP-MIB
+    symbol:
+      name: memory.total
+      OID: 1.3.6.1.4.1.2021.4.5.0
+  - MIB: UCD-SNMP-MIB
+    symbol:
+      name: memory.free
+      OID: 1.3.6.1.4.1.2021.4.6.0
+  - MIB: UCD-SNMP-MIB
+    symbol:
+      name: memTotalSwap
+      OID: 1.3.6.1.4.1.2021.4.3.0
+  - MIB: UCD-SNMP-MIB
+    symbol:
+      name: memAvailSwap
+      OID: 1.3.6.1.4.1.2021.4.4.0
+  - MIB: UCD-SNMP-MIB
+    symbol:
+      name: memTotalFree
+      OID: 1.3.6.1.4.1.2021.4.11.0
+  - MIB: UCD-SNMP-MIB
+    symbol:
+      name: memMinimumSwap
+      OID: 1.3.6.1.4.1.2021.4.12.0
+  - MIB: UCD-SNMP-MIB
+    symbol:
+      name: memShared
+      OID: 1.3.6.1.4.1.2021.4.13.0
+  - MIB: UCD-SNMP-MIB
+    symbol:
+      name: memBuffer
+      OID: 1.3.6.1.4.1.2021.4.14.0
+  - MIB: UCD-SNMP-MIB
+    symbol:
+      name: memCached
+      OID: 1.3.6.1.4.1.2021.4.15.0
+  - MIB: UCD-SNMP-MIB
+    table:
+      OID: 1.3.6.1.4.1.2021.9
+      name: dskTable
+    symbols:
+      - name: dskTotal
+        OID: 1.3.6.1.4.1.2021.9.1.6
+      - name: dskAvail
+        OID: 1.3.6.1.4.1.2021.9.1.7
+      - name: dskUsed
+        OID: 1.3.6.1.4.1.2021.9.1.8
+      - name: dskPercent
+        OID: 1.3.6.1.4.1.2021.9.1.9
+      - name: dskPercentNode
+        OID: 1.3.6.1.4.1.2021.9.1.10
+    metric_tags:
+      - tag: dsk_path
+        column:
+          OID: 1.3.6.1.4.1.2021.9.1.2
+          name: dskPath
+      - tag: dsk_device
+        column:
+          OID: 1.3.6.1.4.1.2021.9.1.3
+          name: dskDevice
+      - column:
+          OID: 1.3.6.1.4.1.2021.9.1.100
```

Comment:
❓ Are we ignoring `101` errorMessage because of cardinality?

Reply:
Yes, indeed, dskErrorMsg is defined as:

```
A text description providing a warning and the space left on the disk.
```

If the "space left on the disk" is used as tag, it might have high cardinality hence by precaution, I'm leaving it out for now.